Sources/Parsers/StringsFileParser.swift

```diff
@@ -10,41 +10,43 @@ import PathKit
 public enum StringsFileParserError: Error, CustomStringConvertible {
   case failureOnLoading(path: String)
   case invalidFormat
+  case duplicateTable(name: String)
 
   public var description: String {
     switch self {
     case .failureOnLoading(let path):
       return "Failed to load a file at \"\(path)\""
     case .invalidFormat:
       return "Invalid strings file"
+    case .duplicateTable(let name):
+      return "Table \"\(name)\" already loaded, cannot add it again"
```

Comment:
The real problem isn't that we _cannot add it again_ (to be fair, we could totally merge the entries form both tables in a single context entry), but more that it wouldn't make sense, because:

* either both tables with the same name come from the same target but for different languages so that would lead to parsing the same entries — just with different translations
* or because they come from different targets (a framework and an app for example) and that wouldn't make sense to merge them in a single huge file while they should be split in different files belonging to their respective targets

So maybe we should find a better wording/explanation for this error? Like this?

`Duplicate table \"\(name)\". Only provide all your strings files for a *single* language (typically your base language) from *one* given target (app or framework).`

Or if you have a better wording don't hesitate :wink: